### PR TITLE
[MISC] Add namespace.

### PR DESCRIPTION
**Pull request by MitraDarja**
I just added the namespace everywhere, I am not sure, if that is actually what we want... Or if there are things that should even be left out of the namespace...

### CODE REVIEW

**Review by SGSSGene — COMMENTED** [review]
I believe you didn't wanted to but the main function into the hibf namespace :-)

**Comment by MitraDarja** [comment]
> I believe you didn't wanted to but the main function into the hibf namespace :-)

Ups. :sweat_smile: Thanks for being so observing! :)

File: include/hibf/build/batch.hpp [hunk 1]
```diff
@@ -33,3 +36,4 @@ inline std::ostream & operator<<(std::ostream & s, batch const & b)
 
     return s;
 }
+} // namespace hibf
```
Comment (eseiler) [inline include/hibf/build/batch.hpp:39]:
```suggestion
}

} // namespace hibf
```

**Review by eseiler — CHANGES_REQUESTED** [review]
Can you add a new line before each closing namespace? Otherwise LGTM

**Review by eseiler — APPROVED** [review]

**Pull request merged**